addons/docs/src/frameworks/angular/angular-to-react.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
+import 'zone.js/dist/zone-error';
+import { document } from 'global';
+import { NgModule, NgModuleRef, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import React from 'react';
+
+const defaultPassProps = (props: any) => props;
+
+export const SELECTOR = 'app-root';
+
+const bootstrapAngularApp = (
+  node: HTMLElement,
+  AppComponentInstance: any
+): Promise<NgModuleRef<any>> => {
+  node.appendChild(document.createElement(SELECTOR));
```

Comment:
Consider allowing users to pass in their own value for `SELECTOR`. When defining angular components, the decorator accepts a selector value. Users may wish to use their own selector value.

```tsx
import { Component } from '@angular/core';

@Component({
  selector: 'app-hero-list', # <-- current code assumes this value === 'app-root' 
  templateUrl: './hero-list.component.html',
})
export class HeroListComponent {
}
```

Reply:
Thanks, that's a good point. 
I started integrating your solution directly in Storybook, hoping for some magic to happen.. 🧙
Still permits to talk about how & where to add this logic, but not good enough for testing.

I ended up making a React side-project and successfully applied your solution there.
Currently looking into injecting metadata, then I'll get into selectors.
I'll let you know once I have news about that, gonna focus on this task for the days to come. 😉